### PR TITLE
[ZEPPELIN-891] Update doap.rdf

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
Update doap.rdf with tlp information

### What type of PR is it?
Improvement

### Todos
* [x] - Update doap.rdf

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-891

### How should this be tested?
N/A

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: doap.rdf [hunk 1]
```diff
@@ -24,19 +24,18 @@
   <Project rdf:about="http://zeppelin.incubator.apache.org/">
```
Comment (minahlee) [inline doap.rdf]: What about this line?

Comment (Leemoonsoo) [inline doap.rdf]: Updated!

**Comment by minahlee** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Thanks @minahlee taking a look. 
Merge it into master if there is no more discussions.

**Pull request closed without merging**